docs/dev/table/sql/gettingStarted.zh.md

```diff
@@ -129,11 +128,11 @@ FROM employee_information
 GROUP BY dep_id;
  {% endhighlight %} 
 
-Such queries are considered _stateful_. Flink's advanced fault-tolerance mechanism will maintain internal state and consistency, so queries always return the correct result, even in the face of hardware failure. 
+这样的查询被认为是 _有状态的_。Flink 的高级容错机制将维持内部状态和一致性，因此即使遇到硬件故障，查询也始终返回正确结果。
 
-## Sink Tables
+## Sink 表
 
-When running this query, the SQL client provides output in real-time but in a read-only fashion. Storing results - to power a report or dashboard - requires writing out to another table. This can be achieved using an `INSERT INTO` statement. The table referenced in this clause is known as a sink table. An `INSERT INTO` statement will be submitted as a detached query to the Flink cluster. 
+当运行此查询时，SQL 客户端实时但是以只读方式提供输出。存储结果，作为报表或仪表板的数据来源，需要写到另一个表。这可以使用 `INSERT INTO` 语句来实现。本节中引用的表称为 sink 表。`INSERT INTO` 语句将作为一个独立查询被提交到 Flink 集群中。
 
  {% highlight sql %}
  INSERT INTO department_counts
```

Comment:
Please remove this blank space.
![image](https://user-images.githubusercontent.com/33114724/103258946-37058300-49d2-11eb-9070-e660eae05f45.png)